api/database/database_manager.py

```diff
@@ -0,0 +1,113 @@
+        VALUES(%s, %s, %s)
+        returning user;""", (username, password, api_key))
+        connection.commit()
+        return "User created"
+    except (Exception, psycopg2.DatabaseError) as error:
+        print(error)
+        return "Error"
+
+
+# NOTE: This function will be implemented soon, I know that the code is doubled below
+
+def authenticate_user(cursor, username, password):
+    cursor.execute("""SELECT * FROM users
+        WHERE username = '{0}'""".format(username))
+    try:
+        user_data = cursor.fetchall()[0]
+        username = user_data[0]
+        db_password = user_data[1]  # hashed password from database
+        if username == username and sha256_crypt.verify(password, db_password):
+            # VERIFIED - username and password are correct
+            return user_data
+        else:
+            return "Username or password is incorrect"
+    except:
+        # NOT VERIFIED - username or password is incorrect
+        return "Username or password is incorrect"
+
+
+def add_exchange(connection, cursor, username, password, exchange_name, exchange_api_key, secret):
+    """ This function adds users' exchange
+        Paramaters
+        ------------
+        connection: connection required - postresql connection
+        cursor: connection required - postresql connection
+        username: str required - username
+        password: str required - password
+        exchange_name: str required - name of and exchange, e.g. binance
+        exchange_api_key: str required - api key from an exchange, e.g. from binance
+        secret: str required - secret api key from an exchange, e.g. from binance
+    """
+    user_data = authenticate_user(cursor, username, password)
+    if user_data != False:
+        try:
+            user_api_key = user_data[2]
+            cursor.execute("""INSERT INTO exchanges
+            VALUES('{0}', '{1}', '{2}', '{3}', '{4}');""".format(exchange_name, username, exchange_api_key,
+                                                                 user_api_key, secret))
+            connection.commit()
+            return "Exchange created"
+        except:
+            return "Error"
+    else:
+        return "Username or password is incorrect"
+
+
+def hash_password(password):
+    """ This function hashes password """
+    hashed_password = sha256_crypt.hash(password)
+    return hashed_password
+
+
+def get_user_trades(cursor, username, password, exchange_name, symbol):
+    """ Getting user trades
+        Paramaters
+        ------------
+        cursor: connection required - postresql connection
+        username: str required - username
+        password: str required - password
+        exchange_name: str required - name of and exchange, e.g. binance
+        symbol: str - symbol of cryptocurrency, e.g. BTC/USDT
+    """
+    user_data = authenticate_user(cursor, username, password)
+    if user_data != False:
```

Comment:
Why we have such weird comparison here?

```suggestion
    if not user_data:
```